lib/event.ts

```diff
@@ -1,8 +1,20 @@
-export function getEventName(
-  name: string | string[] | undefined,
-  eventTitle: string,
-  eventNameTemplate: string | null
-) {
-  if (!name || !(typeof name === "string")) name = ""; // If name is not set or is not of proper type
-  return eventNameTemplate ? eventNameTemplate.replace("{USER}", name) : eventTitle + " with " + name;
+import { TFunction } from "next-i18next";
+
+export type EventNameObjectType = {
+  attendee: string | string[] | undefined;
```

Comment:
```suggestion
  attendee?: string | string[];
```

Maybe even better to name it `attendeeName` as `attendee` indicates it may be an object?

Reply:
the attendeeName is given the type to satisfy the following inferred type
`const name: string | string[] | undefined`
from `router.query`